src/frontend/src/tests/lib/utils/token.utils.spec.ts

```diff
@@ -1,70 +1,75 @@
+import type { TokenStandard } from '$lib/types/token';
 import { getMaxTransactionAmount } from '$lib/utils/token.utils';
 
 const tokenDecimals = 8;
-const tokenId = Symbol('mock');
+const tokenStandards: TokenStandard[] = ['ethereum', 'icp', 'icrc', 'bitcoin'];
 
 const balance = 1000000000n;
 const fee = 10000000n;
 
-vi.mock('$eth/utils/eth.utils', () => ({
-	isSupportedEthTokenId: vi.fn((id: symbol) => id === tokenId)
-}));
-
 describe('getMaxTransactionAmount', () => {
-	it('should return the correct maximum amount for a transaction', () => {
-		const result = getMaxTransactionAmount({
-			balance,
-			fee,
-			tokenDecimals: tokenDecimals,
-			tokenId: tokenId
+	it('should return the correct maximum amount for a transaction for each token standard', () => {
```

Comment:
If you want to test each token separatly, maybe you can implement a `describe.each` instead of iterating in each `it` test? just a suggestion.

Something like:

```
describe('getMaxTransactionAmount', () => {
   describe.each(tokenStandards)("Token standard %i"), (token) => 
```

Reply:
good suggestion! I am not really sure to be honest which one to choose. I imagine the difference is between showing a test for each token or a general test for all token... boh! No strong opinion here, what do you think? If you prefer, I change it

Reply:
No opinion at all, I'm fine with current test. Reading it just made me think at this idea.